src/uikit/SummaryTable/index.js

```diff
@@ -1,52 +1,101 @@
 import React from 'react';
 import styled, { css } from 'react-emotion';
 import PropTypes from 'prop-types';
+import _ from 'lodash';
+import { mq } from 'uikit/BreakpointHelper';
+import { Visible } from 'react-grid-system';
 
-const Table = styled('table')`
-  border: 1px solid ${({ theme }) => theme.greyScale5};
+const SummaryTableWrapper = styled('div')`
+  height: 100%;
   width: 100%;
-  border-collapse: collapse;
+  display: flex;
+  flex-direction: column;
+  border: 1px solid ${({ theme }) => theme.greyScale5};
+
+  > div {
+    border: none;
+    margin-bottom: 5px;
+  }
+
+  ${mq[1]} {
+    flex-direction: row;
+    border: none;
+
+    > div {
+      border: 1px solid ${({ theme }) => theme.greyScale5};
+      margin-right: 20px;
+      align-self: flex-start;
+    }
+
+    > div:last-child {
+      margin-right: 0;
+    }
+  }
 `;
 
-const TableRow = styled('tr')`
-  background-color: ${({ index, theme }) => (index % 2 === 0 ? theme.backgroundGrey : '#fff')};
+const Table = styled('div')`
+  flex: 1;
+  align-sef: flex-start;
+  display: grid;
+  grid-template-columns: auto 1fr;
 `;
 
-const cellBase = props => css`
+const cell = props => css`
   padding: 7px 12px;
   font-size: 13px;
   text-align: left;
-  vertical-align: top;
-`;
-
-const TableCell = styled('td')`
-  ${cellBase};
-  color: ${({ theme }) => theme.greyScale1};
-  font-family: ${({ theme }) => theme.fonts.details};
-  width: 99%;
+  background-color: ${props.index % 2 === 0 ? props.theme.backgroundGrey : '#fff'};
 `;
 
-const TableTitle = styled('td')`
-  ${cellBase};
-  white-space: nowrap;
+const SummaryTitle = styled('div')`
+  ${cell};
   color: ${({ theme }) => theme.secondary};
   font-weight: 600;
   font-family: ${({ theme }) => theme.fonts.default};
-  padding-right: 47px;
+  padding-right: 0;
+`;
+
+const SummaryContent = styled('div')`
+  ${cell};
+  color: ${({ theme }) => theme.greyScale1};
+  font-family: ${({ theme }) => theme.fonts.details};
+  min-width: 100px;
+  overflow-wrap: break-word;
+  word-wrap: break-word;
+  padding-left: 10%;
 `;
 
-const SummaryTable = ({ rows }) => (
-  <Table>
-    <tbody>
-      {rows.map((row, i) => (
-        <TableRow index={i} key={i}>
-          <TableTitle>{row.title}</TableTitle>
-          <TableCell>{row.summary}</TableCell>
-        </TableRow>
-      ))}
-    </tbody>
-  </Table>
-);
+class SummaryTable extends React.Component {
```

Comment:
I think this component is easier to understand as a function, so the `rows` is just fed directly down :P should be a quick switch, any harm? :)